src/components/topics/TopicActivityList.vue

```diff
@@ -0,0 +1,90 @@
+<script setup lang="ts">
+import type { Activity } from '@/model/activity'
+import type { ResolvedFactor, Topic } from '@/model/topic'
+import { useCurrentPageConf } from '@/router/utils'
+import { ActivityList as ActivityListComponent } from '@datagouv/components-next'
+import { useRouter } from 'vue-router'
+
+const props = defineProps<{
+  topic: Topic
+  factors: ResolvedFactor[]
+}>()
+
+const router = useRouter()
+const { pageConf } = useCurrentPageConf()
+
+const getActivityTranslation = (activity: Activity) => {
+  const elementId = getElementId(activity)
+  const activeFactor = getActiveFactor(elementId)
```

Comment:
Not sure about the use of "active" here. I initially thought it had something to do with selection, but it's about whether the factor still exists or not. Maybe the deep-linking stuff in this PR primed me to think "active == selected", but in any case maybe use something less ambiguous, or just `factor` = `getFactor(...)`?